test/conftest.py

```diff
@@ -47,7 +47,7 @@
 SECOND_WIDTH = 640
 SECOND_HEIGHT = 480
 
-max_sleep = 5.0
+max_sleep = 15.0
```

Comment:
Is it really needed? It used to be 20 seconds, but I lowered to 5 as it was super boring to wait more than that in case of real code errors.